core/src/test/resources/application.conf

```diff
@@ -11,3 +11,5 @@ database = {
 messaging {
   mode = "test"
 }
+
+packages.versionFormat = ".+"
```

Comment:
It is not a bug per se, but the `Refine` for `PackageVersion` reads this value from the config, so if you don't set this Config value and try to refine a `PackageId` you will get some other error such as `ClassNotExist` (can't remember the real name) which is really unhelpful.